hugr-py/tests/test_hugr_build.py

```diff
@@ -274,6 +274,28 @@ def test_mono_function(direct_call: bool) -> None:
     validate(mod.hugr)
 
 
+def test_recursive_function() -> None:
+    mod = Module()
+
+    f_recursive = mod.define_function("recurse", [tys.Qubit])
+    f_recursive.declare_outputs([tys.Qubit])
```

Comment:
It would be nice to be able to do 
```suggestion
    f_recursive = mod.define_function("recurse", [tys.Qubit], outputs=[tys.Qubit])
```

is this difficult?

Reply:
Done!